src/update_engine/download_action.cc

```diff
@@ -21,15 +21,13 @@ namespace chromeos_update_engine {
 const size_t kFileWriterBufferSize = 128 * 1024;  // 128 KiB
 
 DownloadAction::DownloadAction(PrefsInterface* prefs,
-                               SystemState* system_state,
                                HttpFetcher* http_fetcher)
     : prefs_(prefs),
-      system_state_(system_state),
       http_fetcher_(http_fetcher),
       writer_(NULL),
       code_(kActionCodeSuccess),
       delegate_(NULL),
-      bytes_received_(0) {}
+      bytes_downloaded_(0) {}
```

Comment:
In the new version of the `BytesReceived` function I use `received` to mean "received just now" and `progress` for "received total`, previously `received` meant "received total". This var is total so I found the old name confusing. Happy to take suggestions for other variable or parameter names, they still are not great but `BytesReceived(uint64_t received_just_now, uint64_t received_total, uint64_t total_expected)` feels overly verbose.